crates/bevy_image/Cargo.toml

```diff
@@ -44,26 +45,26 @@ bevy_reflect = { path = "../bevy_reflect", version = "0.15.0-dev", features = [
 bevy_utils = { path = "../bevy_utils", version = "0.15.0-dev" }
 
 # rendering
-image = { version = "0.25.2", default-features = false }
+image = { version = "0.25", default-features = false }
```

Comment:
These aren't equivalent, and we want at least `0.25.2`. See https://github.com/bevyengine/bevy/pull/14505.

I think there are other instances where this is known to be true, and probably more yet that could be lurking issues if this were merged.

Reply:
Wouldn't a `cargo update` fix that? If the developer doesn't want to do it manually, a post-pull hook would solve that problem.

Reply:
What would be the benefit to specifying a version requirement allowing versions that are known to cause issues?